### PR TITLE
feat: add ignore list for console.error

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

**Review by BrianAmadori — APPROVED** [review]

**Review by pravusjif — APPROVED** [review]

**Comment by menduz** [comment]
Is this the only way we have to do it?. I would merge this only as last resort

File: kernel/packages/unity-interface/setFilteredConsoleError.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const ignoredErrors = ['Asset Bundle download is complete, but no data have been received\n']
+
+export function setFilteredConsoleError() {
+  if (isOverridden) return
+
+  isOverridden = true
+  const defaultConsoleError = window.console.error
+
+  window.console.error = function () {
+    if (!ignoredErrors.includes(arguments[0])) {
```
Comment (menduz) [inline kernel/packages/unity-interface/setFilteredConsoleError.ts:12]: does this work if we write "console.error(null) or console.error(new Error())"?

Comment (pbosio) [reply kernel/packages/unity-interface/setFilteredConsoleError.ts:12]: yeap, just tested that, it works as expected

**Review by D4rWiNSS — APPROVED** [review]

**Comment by menduz** [comment]
The asset bundler now works, it has been a while since I've seen this error. Closing to avoid hacks

**Pull request closed without merging**